lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -101,14 +101,14 @@ class EventTargetImpl {
 
   // https://dom.spec.whatwg.org/#concept-event-dispatch
   // legacyOutputDidListenersThrowFlag optional parameter is not necessary here since it is only used by indexDB.
-  _dispatch(eventImpl, targetOverride /* , legacyOutputDidListenersThrowFlag */) {
+  _dispatch(eventImpl, legacyTargetOverrideFlag /* , legacyOutputDidListenersThrowFlag */) {
     let targetImpl = this;
     let clearTargets = false;
     let activationTarget = null;
 
     eventImpl._dispatchFlag = true;
 
-    targetOverride = targetOverride || targetImpl;
+    const targetOverride = legacyTargetOverrideFlag ? idlUtils.implForWrapper(targetImpl._globalObject._document) : targetImpl;
```

Comment:
Thanks! That's much better. 😁

Reply:
I had to break out the debugger to figure out why `targetImpl._document` wasn't working... darn global proxy vs. global object stuff.